apps/newsletters-api/open-api.yaml

```diff
@@ -80,7 +80,7 @@ paths:
                     type: boolean
                     example: true
                   results:
-                    $ref: '#/components/schemas/Newsletter'
+                    $ref: '#/components/schemas/LegacyNewsletter'
```

Comment:
as above - the new model doesn't existing yet and the path is currently returning `LegacyNewsletter` - will add comments to indicate that this will change.